### PR TITLE
Merge 0.34.0 and 0.33.0

### PR DESCRIPTION
**Pull request by kgeckhart**
Merges in the latest version of YACE mostly to pickup changes related to propagating context through the AWS calls. There's other changes which shouldn't really impact us. 

Full changes from upstream here, https://github.com/nerdswords/yet-another-cloudwatch-exporter/compare/v0.32.0-alpha...v0.34.0-alpha. This was a pain because I never pushed the logging changes upstream which I need to do.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
+# 0.34.0-alpha
+* Update dependencies
+* Add weekly dependabot updates (jylitalo)
+* Add support for regional sts endpoints (matt-mercer)
```
Comment (ferruvich) [inline CHANGELOG.md:4]: 👀 
@cristiangreco I don't remember exactly what was the problem we had trying regional STS endpoints, but the blocker was before hitting the exporter, right?

**Review by ferruvich — APPROVED** [review]
🚀

File: CHANGELOG.md (hunk 1 above)
Comment (cristiangreco) [reply CHANGELOG.md:4]: I think our problem was around "job validation". Anyway chatted with Kyle yesterday about this specific change and it shouldn't impact us as we use a custom `session`.

File: cmd/yace/main.go [hunk 2]
```diff
@@ -207,15 +215,15 @@ func (s *scraper) scrape(ctx context.Context, cache exporter.SessionCache) {
 	defer sem.Release(1)
 
 	newRegistry := prometheus.NewRegistry()
-	for _, counter := range []prometheus.Counter{exporter.CloudwatchAPICounter, exporter.CloudwatchAPIErrorCounter, exporter.CloudwatchGetMetricDataAPICounter, exporter.CloudwatchGetMetricStatisticsAPICounter, exporter.ResourceGroupTaggingAPICounter, exporter.AutoScalingAPICounter, exporter.ApiGatewayAPICounter, exporter.TargetGroupsAPICounter} {
+	for _, counter := range []prometheus.Counter{exporter.CloudwatchAPICounter, exporter.CloudwatchAPIErrorCounter, exporter.CloudwatchGetMetricDataAPICounter, exporter.CloudwatchGetMetricStatisticsAPICounter, exporter.ResourceGroupTaggingAPICounter, exporter.AutoScalingAPICounter, exporter.ApiGatewayAPICounter, exporter.TargetGroupsAPICounter, exporter.DmsAPICounter} {
```
Comment (cristiangreco) [inline cmd/yace/main.go]: I _think_ we're missing `exporter.Ec2APICounter` here (should be 10 in total).

**Review by cristiangreco — APPROVED** [review]
LGTM, just one minor comment about a counter.

File: CHANGELOG.md (hunk 1 above)
Comment (ferruvich) [reply CHANGELOG.md:4]: Thanks! 🙌

File: cmd/yace/main.go (hunk 2 above)
Comment (kgeckhart) [reply cmd/yace/main.go]: Great catch! Perhaps I should do a follow up to export an array which contains all of them instead to help avoid this.

**Pull request merged**